GruntFile.js

```diff
@@ -37,6 +38,18 @@ module.exports = function(grunt) {
                 }
             }
         },
+        replace: {
+            version: {
+                src: [
+                    'package.js'
+                ],
+                overwrite: true,
+                replacements: [{
+                    from: oldVersion,
+                    to: version
```

Comment:
Actually the version up there I think was from a merge of older stuff and shouldn't be there... the version should come from the _pkg.version, i.e. from the package.json.   The package.js can probably do the same thing, no?